Automattic/woocommerce-payments
# Update references to woocommerce_payments_server

Fixes #9823

#### Changes proposed in this Pull Request

* Updates server container prefix from `woocommerce_payments_server` to `transact_platform_server`
* Updates server repo folder name from `wcp-server` to `transact-platform-server`
* Updates the GitHub secret `WCP_SERVER_REPO` to `TRANSACT_PLATFORM_SERVER_REPO`
* Updates the env variable `WCP_SERVER_BRANCH` to `TRANSACT_PLATFORM_SERVER_BRANCH`
* Updates docs where required

#### Testing instructions

* Follow the instructions on https://github.com/Automattic/woocommerce-payments/tree/update/server-container-name/tests/e2e to set up the local e2e environment and confirm that the environment is working as expected.

-------------------


**Post merge**
